WeakAuras/Transmission.lua

```diff
@@ -1109,13 +1234,6 @@ function WeakAuras.ShowDisplayTooltip(data, children, icon, icons, import, compr
 
     local RegularGetData;
     if(children) then
-      data.controlledChildren = {};
-      for index, childData in pairs(children) do
-        if(compressed) then
-          DecompressDisplay(childData);
-        end
-        data.controlledChildren[index] = childData.id;
```

Comment:
This no longer happens.

Reply:
Can you clarify? Do you mean to say that the code never executed anyways (that feedback wouldn’t make sense since this hunk was removed)?

Reply:
The actual line I commented on was not moved above. I'm not sure whether that is actually a problem.

Reply:
It was a no-op, since both import and update handlers will ensure that controlledChildren has the correct children.